### PR TITLE
Add and update some things (See notes)

### PR DESCRIPTION
**Pull request by cookta2012**
* Add some color definitions to common/ui.lua aswell
* Add some code for screens/character-selection.lua
* Updated common/globals.lua to include button hover flag to fix dragging on buttons

### CODE REVIEW

File: common/globals.lua [hunk 1]
```diff
@@ -8,6 +8,7 @@ RegionDefs = require('common/region-defs')
 ResourceDefs = require("common/resource-defs")
 BasePath = abyss.getConfig("#Abyss", "BasePath")
 MPQRoot = abyss.getConfig("OpenDiablo2", "MPQRoot")
+IsOnButton = false
```
Comment (essial) [inline common/globals.lua:11]: I'm worried about this being a global.. Can we move it to a specific screen?

File: config.ini [hunk 2]
```diff
@@ -19,5 +19,5 @@ Resurrected=0
 MPQs=patch_d2.mpq,d2exp.mpq,d2xmusic.mpq,d2xtalk.mpq,d2xvideo.mpq,d2data.mpq,d2char.mpq,d2music.mpq,d2sfx.mpq,d2video.mpq,d2speech.mpq
 CASCs=./D2RData
 Language=auto
-SkipStartupVideos=0
+SkipStartupVideos=1
```
Comment (essial) [inline config.ini]: Whoops?

File: common/globals.lua (hunk 1 above)
Comment (cookta2012) [reply common/globals.lua:11]: This kinda needs to be global

File: common/ui.lua [hunk 3]
```diff
@@ -10,14 +11,33 @@ function InitUI()
         imgCheckbox = imgButtonShortBlank
     end
 
+    TextColor = {
+        --blackHalfOpacity = 0x0000007f
+        LightBrown = { R = 188, G = 168, B = 140 },
+        LightGreen = { R = 24, G = 255, B = 0 },
+        Black = { R = 255, G = 255, B = 255 },
```
Comment (DarthGandalf) [inline common/ui.lua]: This is white, not black. Black on buttons come from the "multiply" blend mode

File: screens/map-engine-test.lua [hunk 4]
```diff
@@ -120,19 +120,20 @@ function MapEngineTest:initialize()
...
             self.startMouseX = x
             self.startMouseY = y
         end
     end)
     self.inputListener:onMouseButton(function(button, isPressed)
+        if IsOnButton then return end
         if button == 1 then -- Left mouse button
             if isPressed then
-                self.isMouseDrag = true
+                isMouseDrag = true
```
Comment (DarthGandalf) [inline screens/map-engine-test.lua]: this is global now

File: common/ui.lua (hunk 3 above)
Comment (cookta2012) [reply common/ui.lua]: uugh

Comment (cookta2012) [reply common/ui.lua]: fixed

**Review by essial — APPROVED** [review]

**Pull request merged**